### DISCUSSION TITLE
Radxa X4 support?

### DISCUSSION THREAD

**Discussion by alpham0d3x** [Ysurac/openmptcprouter#3468] [Ideas]
Hi there, is support for the Radxa X4 (Intel N100 chipset) planned at some point in the future? I think it's a great value for a board in the same price category as the Raspberry Pi 5. Cheers, David

**Comment by Ysurac** [comment]
Hi, as it's Intel N100, it doesn't work with snapshot image using kernel 6.6 and I think UEFI ?
